pkg/buildpipeline/pipeline.go

```diff
@@ -90,9 +107,11 @@ type Unstash struct {
 type StageOptions struct {
 	RootOptions `yaml:",inline"`
 
-	Stash     Stash   `yaml:"stash,omitempty"`
-	Unstash   Unstash `yaml:"unstash,omitempty"`
-	Workspace *string `yaml:workspace,omitempty`
+	// TODO: Not yet implemented in build-pipeline
+	Stash   Stash   `yaml:"stash,omitempty"`
+	Unstash Unstash `yaml:"unstash,omitempty"`
+
+	Workspace *string `yaml:"workspace,omitempty"`
```

Comment:
Ah, good to know that a blank line will keep gofmt from trying to align the previous blocks with this line.